skopeo_cidev/setup.sh

```diff
@@ -9,7 +9,6 @@ set -e
 declare -a req_vars
 req_vars=(\
     REG_REPO
-    REG_COMMIT
```

Comment:
Also delete the definition from `skopeo_cidev/Containerfile`?

Reply:
yup, it's removed, see the Containerfile diff. Unless you meant something else.

Reply:
Yes, that’s what I meant.